examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -68,15 +66,14 @@
   public void testE2EWordCount() throws Exception {
     PipelineOptionsFactory.register(WordCountITOptions.class);
     WordCountITOptions options = TestPipeline.testingPipelineOptions().as(WordCountITOptions.class);
+
+    IOChannelUtils.registerStandardIOFactories(options);
```

Comment:
Do they need to be registered as e.g. WordCountITOptions? If not, moving to TestPipeline seems to make sense to me.